src/Components/FoodBlock/FoodBlock.tsx

```diff
@@ -16,23 +40,70 @@ const FoodBlock = () => {
         padding={'1rem'}
         justifyContent={'space-between'}
       >
-        {foodProgramm.map((title) => {
+        {foodProgram.map((title) => {
           return <ProgrammFoodBlock title={title} key={title} />
         })}
       </Stack>
       <Stack flexDirection={'row'} width={'100%'} height={'65%'}>
         <Stack width={'70%'} flexDirection={'row'}>
-          <Stack width={'33%'} alignItems={'center'} borderRight={0.25}>
+          <Stack width={'33%'} alignItems={'center'} borderRight={0.25} padding={'0.25rem'}>
             <Typography variant="h6">{t('t-breakfast')}</Typography>
+            {breakfastFoods.map((item, index) => {
+              return (
+                <FoodChipWrapper
+                  key={index}
+                  variant="outlined"
+                  color="success"
+                  label={item.nameFood}
+                  id={item.id}
+                  onDelete={() => deleteHandler(item.id)}
```

Comment:
you can pass deleteHandler as a prop to Chip component and inside this component create handler